src/lua/zencode_when.lua

```diff
@@ -27,7 +27,7 @@ When("done", function() end)
 
 
 local function _is_found(el)
-    return ACK[el] and (luatype(ACK[el]) == 'table' or #ACK[el] ~= 0)
+    return ACK[el] and (luatype(ACK[el]) == 'table' or tonumber(ACK[el]) or #ACK[el] ~= 0)
```

Comment:
we have the safer `table_size()` in place of the `#` overload which may be convenient to use here.

Reply:
The # is used to check that `ACK[el]` is not an empty string. Empty table are accepted since it return true at the first check and thus the others checks are not performed. Do we want to modify it and return as not found also empty tables?